common/djangoapps/geoinfo/tests/test_middleware.py

```diff
@@ -25,9 +25,7 @@ def setUp(self):
         self.request_factory = RequestFactory()
         self.patcher = patch.object(pygeoip.GeoIP, 'country_code_by_addr', self.mock_country_code_by_addr)
         self.patcher.start()
-
-    def tearDown(self):
-        self.patcher.stop()
+        self.addCleanup(self.patcher.stop)
```

Comment:
You can just remove the `tearDown` here and move the `addCleanup()` call to `setUp`, correct?